### ISSUE TITLE
[stg] Buggy ui creating process 

### ISSUE THREAD

**Issue by NateWilliams2** [vocdoni/vocdoni-manager-frontend#13]
When creating a process, I get this pop-up:
<img width="538" alt="Screen Shot 2021-02-16 at 2 39 56 PM" src="https://user-images.githubusercontent.com/43301705/108119553-980e1680-7065-11eb-9abf-0f134db28098.png">

Then I see this window:
<img width="543" alt="Screen Shot 2021-02-16 at 2 39 59 PM" src="https://user-images.githubusercontent.com/43301705/108119520-8a589100-7065-11eb-9201-bc9b334a859f.png">

After a few seconds, however, I return to the first image, and continuously cycle between the two. From the explorer and app it doesn't seem like the process is ever created.

**Comment by elboletaire** [comment]
There may be some error underneath that is not properly displayed. Could you try it again with the console open and report me anything you see there (specially the stuff around the time of the creation process, no need for the gw check errors)? Also, could you describe how did you create the process? 

Also, details like the kind of recipients (either census/all members/spreadsheet) and other selected options would be welcome.

**Comment by NateWilliams2** [comment]
This is an example of the details I'm using:
<img width="496" alt="Screen Shot 2021-02-17 at 8 55 03 AM" src="https://user-images.githubusercontent.com/43301705/108222040-069eb300-70fe-11eb-8d99-b45fa1b4647a.png">

As you can see, it's an all-members process, with all default options other than real-time results.

this error appears in the console: 

```
Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=Error%3A%20The%20method%20is%20not%20available%20in%20the%20Gateway's%20supported%20API's%20(addCensus)&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    NextJS 53
        Ci
        _i
        Ma
        ko
        xu
        wu
        du
        Xl
        unstable_runWithPriority
        $l
        Xl
        Yl
        ou
        enqueueSetState
        setState
        add
        notice
        r
        L
        r
        open
        t
        e
        s
        _invoke
        E
        n
        u
        promise callback*n
        s
        promise callback*n
        s
        i
        i
        value
        u
        F
        o
        p
        y
        y
        at
        lt
        ot
        pt
        L
        W
        Jt
        Gt
        unstable_runWithPriority
        $l
        F
        Yt
```

**Comment by elboletaire** [comment]
Fixed in 494444d39194d7f8f6128f5773425dcab7478e08
